__tests__/e2e/docker-compose.yml

```diff
@@ -39,7 +39,7 @@ services:
       args: *common-variables
       dockerfile: ./__tests__/e2e/Dockerfile
 
-    command: /bin/sh -c "./wait-for-command.sh -t 15 -c 'nc -z datatools-ui 9966' && sleep 25 && yarn test-end-to-end"
+    command: /bin/sh -c "./wait-for-command.sh -t 15 -c 'nc -z datatools-ui 9966' && sleep 40 && yarn test-end-to-end"
```

Comment:
I've changed my mind and I actually think we should revert this change after all. e2e tests seem to be fickle but not because of this.

Reply:
Yeah agreed, I'll revert rn